app/pulp/app/models/task.py

```diff
@@ -35,6 +35,30 @@ class ReservedResource(Model):
     worker = models.ForeignKey("Worker", on_delete=models.CASCADE, related_name="reservations")
 
 
+class WorkerManager(models.Manager):
+    def get_unreserved_worker(self):
```

Comment:
not a standard style request, but consider adding a line of whitespace between the class definition and the first method.